### PR TITLE
Added a footer

### PR DESCRIPTION
**Pull request by Subhan636**
I just make logo look more clear and add some space so it can look more neat (change as you want). also added footer page.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -39,4 +39,155 @@
...
+  transition: background-color 0.3s ease;
+}
+
+button:hover {
+  background: var(--secondary-color);
+}
+
+/* Utility classes */
+.text-center {
+  text-align: center;
```
Comment (gtaEPIC) [inline src/App.css]:
```suggestion
```
Pretty sure this is just making the page look worse. It's making things much more bubbly and mis-aligned and is just adding stuff to something that doesn't need to be changed.

**Review by gtaEPIC — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,40 @@
...
+      {/* Include the TicketDetails component */}
+      <TicketDetails 
+        priority={ticketInfo.priority} 
+        username={ticketInfo.username} 
+        resolution={ticketInfo.resolution} 
+      />
+
+      {/* Optionally, you can render ticket data here */}
+      {/* ... */}
+    </div>
```
Comment (gtaEPIC) [inline src/App.js:37]: Wrong File that this component is being applied to.
Also sus chatGPT usage.

File: src/components/TicketDetails.js
Comment (gtaEPIC) [inline src/components/TicketDetails.js:1]: This is not what #33 was requesting.
It meant to add on to the editing page, not it's own component.
Also you shouldn't be using chatGPT

**Review by gtaEPIC — CHANGES_REQUESTED** [review]
This is not what the issue was referring to.
Also don't use chatGPT, this should be an easy course, and if you have to use chatGPT to do your work then you should be asking for help.

**Comment by Subhan636** [comment]
I was confused about that issue so i used blackbox !!! I will change it once i am off work

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by gtaEPIC** [comment]
This pull request is being closed as the group has decided not to have a footer

**Pull request closed without merging**